src/validators/typed_dict.rs

```diff
@@ -257,39 +236,39 @@ impl Validator for TypedDictValidator {
                             continue;
                         }
 
-                        if self.forbid_extra {
-                            errors.push(ValLineError::new_with_loc(
-                                ErrorType::ExtraForbidden,
-                                value,
-                                raw_key.as_loc_item(),
-                            ));
-                            continue;
-                        }
-
+                        // Unknown / extra field
                         let py_key = either_str.as_py_string(py);
-                        if let Some(ref mut fs) = fields_set_vec {
-                            fs.push(py_key.into_py(py));
-                        }
-
-                        if let Some(ref validator) = self.extra_validator {
-                            match validator.validate(py, value, &extra, slots, recursion_guard) {
-                                Ok(value) => {
-                                    output_dict.set_item(py_key, value)?;
+                        match self.extra_behavior {
+                            ExtraBehavior::Forbid | ExtraBehavior::Unset => {
```

Comment:
I think the default is not to forbid extra.

Reply:
Changed thanks